microsoft-edge/webview2/concepts/process-related-events.md

```diff
@@ -0,0 +1,636 @@
+###### Types of processes
+
+The processes in the _WebView2 Process Group_ can be associated to one or many WebView2 controls in your application. For example:
+
+* **Main browser process.** There is a single main browser process in the _WebView2 Process Group_. Every WebView2 control with the same environment configuration will share this process.
+
+* **Renderer process.** Renderer processes can be associated many-to-many with the WebView2 controls in your application. The details about how this works depend on many factors, including what sites are loaded in the WebView2 control (main frame and subframes), system resources, and runtime configuration. A single renderer process in the _WebView2 Process Group_ can be associated with one or many WebView2 controls in your app.
+
+* **GPU process.** There is a single GPU process in the _WebView2 Process Group_. It is associated with all WebView2 controls using this _WebView2 Process Group_.
+
+* **Utility processes.** Utility processes host one or more _services_ in the _WebView2 Process Group_. Each utility process supports the entire _WebView2 Process Group_ and is thus associated with all WebView2 controls using this _WebView2 Process Group_.
+
+* **Other processes.** Most processes in the _WebView2 Process Group_ are associated to all WebView2 controls using it and will raise `ProcessFailed` to each control.
+
+> [!NOTE]
+> Do not rely on the details about how processes are associated to each WebView2 control, as they are part of the evolving Chromium architecture and subject to change even due to configuration and system conditions. For more information see [Process Model](process-model.md).
+
+When a process in the _WebView2 Process Group_ exits unexpectedly (for example, due to a crash), every WebView2 control **associated** to it will raise a `ProcessFailed` event. You can use the process failure details to decide how to handle each case; see [Gather process failure details](#gather-process-failure-details) above.  The following sections give some examples of what these cases can be:
+
+* [The main browser process has exited unexpectedly](#the-main-browser-process-has-exited-unexpectedly)
+* [A process rendering content in the WebView2 control has exited unexpectedly](#a-process-rendering-content-in-the-webview2-control-has-exited-unexpectedly)
+* [The GPU process has exited unexpectedly](#the-gpu-process-has-exited-unexpectedly)
+* [A utility process has exited unexpectedly](#a-utility-process-has-exited-unexpectedly)
+* [Any other process has exited unexpectedly](#any-other-process-has-exited-unexpectedly)
+
+
+<!-- ---------- -->
+###### The main browser process has exited unexpectedly
+
+All the WebView2 controls in your application using the same environment configuration will receive the `ProcessFailed` event with:
+* **Failure kind:** `BrowserProcessExited`.
+* **Failure reason:** any, except `Unresponsive` and `LaunchFailed`.
+
+##### [.NET/C#](#tab/dotnetcsharp)
```

Comment:
Do we need to repetitively include the APIs and enums that we've already covered?

Reply:
might replace some tab-sets by a link to the (expanded) Process Management section of "Overview of APIs" article

Reply:
streamlined api ref links